hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java

```diff
@@ -1184,6 +1189,10 @@ public void testTrash() throws Exception {
     try (FSDataOutputStream stream = fs.create(path)) {
       stream.write(1);
     }
+    Assert.assertTrue(trash.getConf().getClass(
+        "fs.trash.classname", TrashPolicyDefault.class).
```

Comment:
This should be 

trash.getConf().getClass(
        "fs.trash.classname", TrashPolicyOzone.class, TrashPolicy.class) ?

Reply:
Referring to the following lines of code

https://github.com/apache/hadoop/blob/6f5a36c13c4d2eb24e207aa99a9781c0be11660b/hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/TestTrash.java#L643-L642

Reply:
done